tpot/built_in_operators.py

```diff
@@ -52,19 +52,11 @@ def transform(self, X, y=None):
 
         X_transformed = np.copy(X)
 
-        non_zero = np.apply_along_axis(
-            lambda row: np.count_nonzero(row),
-            axis=1,
-            arr=X_transformed
-        )
-        zero_col = np.apply_along_axis(
-            lambda row: (n_features - np.count_nonzero(row)),
-            axis=1,
-            arr=X_transformed
-        )
-
-        X_transformed = np.insert(X_transformed, n_features, non_zero, axis=1)
-        X_transformed = np.insert(X_transformed, n_features + 1, zero_col, axis=1)
+        non_zero = np.reshape(np.count_nonzero(X_transformed, axis=1), (-1, 1))
+        zero_col = np.reshape(n_features - np.count_nonzero(X_transformed, axis=1), (-1, 1))
```

Comment:
Please rewrite this line to reuse the `non_zero` values calculated just above it.